src/mmrelay/db_utils.py

```
@@ -565,77 +731,127 @@ def _initialize(cursor: sqlite3.Cursor) -> None:
+                        )
+                    else:
+                        cursor.execute(
+                            _INSERT_OR_IGNORE_MESSAGE_MAP_FROM_LEGACY_WITHOUT_MESH_SQL
+                        )
+            if temp_exists:
+                try:
+                    cursor.execute(_INSERT_OR_IGNORE_MESSAGE_MAP_FROM_TEMP_SQL)
+                    logger.info(
+                        "Merged rows from temporary table into rebuilt message_map"
+                    )
+                except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)